framework/utils/randomUtils.py

```diff
@@ -96,6 +96,7 @@ def randomSeed(value):
     npStochEnv = np.random.RandomState(value)
   else:
     crowStochEnv.seedRandom(value) #this has a void return, right?
+  print('randomUtils: Global random number seed has been changed to',value)
```

Comment:
Is this print needed?  Can it be switched to a debug message?

Reply:
No message handler is available in the utilities, unfortunately.

Reply:
Okay, then it is fine as is.